crates/parser/src/lib.rs

```diff
@@ -16,6 +16,12 @@ use ast::{Assertion, AssertionKind, Comparison, NodeName, ParaId, TestDefinition
 #[cfg(test)]
 mod tests;
 
+enum ScriptType {
+    JS,
+    TS,
+    SH,
+}
```

Comment:
How about being more explicit ?

```suggestion
enum ScriptType {
    Javascript,
    Typescript,
    Shell,
}
```